packages/react/src/primitives/StepperField/StepperField.tsx

```diff
@@ -43,6 +43,7 @@ const StepperFieldPrimitive: Primitive<StepperFieldProps, 'input'> = (
     variation,
     testId,
 
+    alignSelf, // @TODO: remove custom destructuring for 3.0 release
```

Comment:
When is the 3.0 release?

Reply:
I know, didn't want to update all the other lines since they also have the wrong release number 😱. This line will be completely removed in 5.x